controllers/authController.js

```diff
@@ -0,0 +1,64 @@
+import jwt from 'jsonwebtoken';
+import AppError from '../errors/AppError';
+import * as UserModel from '../models/UserModel';
+
+const logger = require('../utils/logger')('logController');
+
+
+const register = async (req, res, next) => {
+    logger.log('debug', 'register: %j', req.body);
+    const { username, email, hashedPassword, name, surname, dateOfBirth, level } = req.body;
```

Comment:
Instead of deconstructing all values separately just get the body from req and use it.